src/main/webapp/app/modules/upload/upload-page.tsx

```diff
@@ -109,9 +109,21 @@ export class UploadPage extends React.Component<IUploadPageProp, IUploadState> {
 
   fileValidateTypeDetectType = (source, type) =>
     new Promise((resolve, reject) => {
+      type = this.handleUndefinedFileType(source, type);
       resolve(type);
     });
 
+  handleUndefinedFileType = (source, type) => {
```

Comment:
Does it handle undefined on any system/browser or for a specific one?

Reply:
It handles every system/browser.